.csslintrc

```diff
@@ -0,0 +1,28 @@
+{
+  "adjoining-classes": false,
+  "box-sizing": false,
+  "box-model": false,
+  "bulletproof-font-face": 2,
+  "display-property-grouping": 2,
+  "duplicate-background-images": 2,
+  "duplicate-properties": 2,
+  "empty-rules": 2,
+  "floats": 2,
+  "outline-none": 2,
+  "font-faces": 2,
+  "ids": 2,
+  "import": 2,
+  "important": 2,
+  "known-properties": 2,
+  "qualified-headings": 2,
+  "regex-selectors": 2,
+  "shorthand": 2,
+  "star-property-hack": 2,
+  "text-indent": 2,
```

Comment:
Personally I'm not bothered by text-indent being used in this way, it's a very rarely used hack and I doubt it will end up in the toolkit.